web/src/rest/publish.ts

```diff
@@ -156,6 +156,10 @@ const publishRest = new Vue({
       const { data } = await client.get('users/search/?', { params: { username } });
       return data;
     },
+    async publish(identifier: string): Promise<Version> {
+      const { data } = await client.post(`dandisets/${identifier}/versions/draft/publish/`);
```

Comment:
What (if anything) should we do if the POST fails?

Reply:
I'd expect some kind of :warning: GUI alert :warning:, but right now we have no error notifications anywhere in the app. Eventually we should think about failure paths, but the time is not yet nigh.